metaflow/sidecar_messages.py

```diff
@@ -4,21 +4,26 @@
 # Unfortunately we can't use enums because they are not supported
 # officially in Python2
 class MessageTypes(object):
-    SHUTDOWN, LOG_EVENT = range(1, 3)
+    INVALID, START, UPDATE_CONTEXT, SHUTDOWN, LOG_EVENT = range(1, 6)
 
 
 class Message(object):
     def __init__(self, msg_type, payload):
         self.msg_type = msg_type
         self.payload = payload
 
-    def serialize(self):
+    def serialize(self, clear_previous=False):
```

Comment:
`clear_previous`? Wherever you are using the output of (erstwhile)`serialize`, can't you just prepend a new line right there rather than bleeding in the logic here?

Reply:
Yep, can do that. I thought this was a bit cleaner but happy to move it out.